stdlib/math.jou

```diff
@@ -1,40 +1,112 @@
+# Math lib
+#TODO: add test
+
+# Other functions
+# Returns the absolute value of x.
 declare abs(x: int) -> int
 declare llabs(x: long) -> long
 declare fabs(x: double) -> double
+# Returns x*y+z.
+declare fma(x: double, y: double, z: double) -> double
 
+# Rounding and remainder functions
+# Rounds x upward, returning the smallest integral value that is not less than x.
 declare ceil(x: double) -> double
+# Rounds x downward, returning the largest integral value that is not greater than x.
 declare floor(x: double) -> double
+# Returns the floating-point remainder of numer/denom (rounded towards zero):
+# fmod = numer - tquot * denom
+declare fmod(numer: double, denom: double) -> double
+# Rounds x toward zero, returning the nearest integral value that is not larger in magnitude than x.
+declare trunc(x: double) -> double
+# Returns the integral value that is nearest to x, with halfway cases rounded away from zero.
 declare round(x: double) -> double
+declare lround(x: double) -> long
+# Rounds x to an integral value,
+declare rint(x: int) -> double
+declare lrint(x: long) -> double
+# Rounds x to an integral value
+declare nearbyint(x: double) -> double
```

Comment:
What is the difference between `round()` and `nearbyint()`? Why do we need both?